src/Model/Email/HoldEmail.php

```diff
@@ -26,9 +27,12 @@ public function getFromAddress()
      */
     public function getToAddress()
```

Comment:
Nitpicking the name of this method. Would prefer it to be getToEmailAddress() for clarity. I know it would be longer.